inc/api-dashboard.php

```
@@ -103,20 +103,152 @@ public function get_zen_activity($request) {
+            ],
+            'message' => 'GamiPress not active',
+        ];
+
+        if (!function_exists('gamipress_get_user_points')) {
+            return new WP_REST_Response($default_response, 200);
+        }
+
+        $points_type = djz_get_gamipress_points_type_slug();
+        $points = (int) gamipress_get_user_points($user_id, $points_type);
+
+        $tiers_payload = $this->get_rank_tiers();
```

PHPMD (2.15.0)

The variable $tiers_payload is not named in camelCase. (undefined)

(CamelCaseVariableName)